x-pack/plugins/enterprise_search/public/applications/app_search/components/credentials/credentials_list/key.test.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { shallow } from 'enzyme';
+import { EuiButtonIcon } from '@elastic/eui';
+
+import { Key } from './key';
+
+describe('Key', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  const props = {
+    copy: jest.fn(),
+    toggleIsHidden: jest.fn(),
+    isHidden: true,
+    text: '*****',
```

Comment:
This confused me slightly on first glance because we changed these to bullets - should we change this to actual text so it's clearer the text is what's being hidden?

```suggestion
    text: 'some-api-key',
```